packages/docs/src/examples/data-tables/customizeHeaders.vue

```diff
@@ -1,49 +1,18 @@
 <template>
   <v-data-table
-    v-model="selected"
     :headers="headers"
     :items="desserts"
-    :options.sync="options"
-    show-select
-    item-key="name"
-    hide-default-header
     class="elevation-1"
   >
-    <template v-slot:header="{ props, on }">
-      <thead>
-        <tr>
-          <th>
-            <!-- <v-checkbox
```

Comment:
For what is this comment needed?

Reply:
Not sure why you're commenting on removed code?